publication-commons/src/main/java/no/unit/nva/publication/model/storage/FileDao.java

```diff
@@ -40,9 +42,10 @@
 
 @JsonTypeName(FileDao.TYPE)
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
-public final class FileDao extends Dao implements DynamoEntryByIdentifier {
+public final class FileDao extends Dao implements DynamoEntryByIdentifier, JoinWithResource {
```

Comment:
Main change for file. JoinWithResource interface is implementation of PK2 for Dao's